experiment_config/coref.json

```diff
@@ -0,0 +1,99 @@
+{
+  "dataset_reader": {
+    "type": "coref",
+    "token_indexers": {
+      "tokens": {
+        "type": "single_id",
+        "lowercase_tokens": false
+      },
+      "token_characters": {
+        "type": "characters"
+      }
+    },
+    "max_span_width": 10
+  },
+  "train_data_path": "/net/efs/aristo/allennlp/coref/ontonotes-coref/train.english.v4_gold_conll",
```

Comment:
Please use http paths to our S3 bucket rather than EFS paths.  These are not portable outside AI2.

I'm happy to help upload this data to our S3 buckets, but it's really easy.  For example, `aws s3 cp file s3://allennlp/datasets/coref/ --acl=public-read`

Reply:
Unfortunately we can't distribute this data. I've modified the paths in the same way as we have done for the SRL model though, so we can add some instructions on the website.